invenio_cli/cli/packages.py

```diff
@@ -67,11 +67,26 @@ def install(cli_config, packages, skip_build):
 @pass_cli_config
 def outdated(cli_config):
     """Show outdated Python dependencies."""
-    raise click.UsageError("Not supported yet.")
+    steps = PackagesCommands.outdated_packages()
+
+    on_fail = "Some of the packages need to be updated."
+    on_success = "All packages are up to date."
+
+    run_steps(steps, on_fail, on_success)
 
 
 @packages.command()
+@click.argument("packages", nargs=-1, type=str)
 @pass_cli_config
-def update(cli_config):
-    """Update a single Python python package."""
-    raise click.UsageError("Not supported yet.")
+def update(cli_config, packages=None):
+    """Update all or some Python python packages."""
+    if len(packages) >= 1:
```

Comment:
if above does not allow defaults... you need to check for `if packages and len(pacakges) >= 1` otherwise, if packages is `None` it will throw an exception... `TypeError: object of type 'NoneType' has no len()`